src/components/PlayerView/playerViewConfig.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Configures the visual presentation and behaviour of the `PlayerView`.
+ */
+export interface PlayerViewConfig {
+  /**
+   * Configures the visual presentation and behaviour of the Bitmovin Player UI.
+   * A `WebUiConfig` can be used to configure the default Bitmovin Player Web UI.
+   *
+   * Default is `WebUiConfig`.
+   *
+   * Limitations:
+   * Configuring the `uiConfig` only has effect if the `StyleConfig.userInterfaceType` is set to `Bitmovin`.
+   */
+  uiConfig: UiConfig;
+}
+
+/**
+ * Configures the visual presentation and behaviour of the Bitmovin Player UI.
+ */
+export interface UiConfig {}
+
+/**
+ * Configures the visual presentation and behaviour of the Bitmovin Web UI.
+ */
+export interface WebUiConfig extends UiConfig {
+  /**
+   * Whether the Bitmovin Web UI will show playback speed selection options in the settings menu.
+   * Default is `true`.
+   */
+  playbackSpeedSelectionEnabled?: boolean;
+}
```

Comment:
This is not ideal as there is no way in typescript to detect the type of an object anonymously implementing an interface, hence we will have to "detect" the ui config type by checking for specific properties once we have more. 
I am open for suggestions if you have an idea how to improve this.